app/src/main/java/ca/carleton/gcrc/n2android_mobile1/connection/ConnectionSyncProcess.java

```diff
@@ -549,9 +692,7 @@ private SubmissionStatus getSubmissionStatusForDocument(JSONObject document) thr
     private boolean IsDocumentDeletedOnRemote(JSONObject doc) throws Exception {
         SubmissionStatus submissionStatus = getSubmissionStatusForDocument(doc);
 
-        boolean requiresDeletion = submissionStatus != SubmissionStatus.WaitingForApproval && !isRequiresSubmission(doc);
-
-        return requiresDeletion;
+        return submissionStatus != SubmissionStatus.WaitingForApproval && !isRequiresSubmission(doc);
```

Comment:
do we care about `SubmissionStatus.Declined`?

Reply:
No, if the document was submitted and they rejected the edits, this should delete the document (Since it was already deleted). If they re-create the doc later, it'll be re-added in a future step.